wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -950,3 +955,8 @@ extension UserClient {
     }
 
 }
+
+public extension Notification.Name {
+    // This notification is used to notify of end-to-end identity certificate changes
+    static let e2eiCertificateChanged = NSNotification.Name("E2EICertificateStatusChanged")
```

Comment:
Can we move the notification to `E2EI+Notifications` struct instead of the extension?